File: src/co/aospa/sense/camera/callables/CallableReturn.java
package co.aospa.sense.camera.callables;

public class CallableReturn {
    public Exception exception;
    public Object value;

    public CallableReturn(Object value) {
        this.value = value;
        exception = null;
    }

    public CallableReturn(Exception exc) {
        exception = exc;
        value = null;
    }

}

File: src/co/aospa/sense/camera/callables/CallableReturn.kt
package co.aospa.sense.camera.callables

import java.lang.Exception

class CallableReturn {
    @JvmField
    var exception: Exception?
    @JvmField
    var value: Any?

    constructor(obj: Any?) {
        value = obj
        exception = null
    }

    constructor(e: Exception?) {
        exception = e
        value = null
    }
}